satviz-consumer-native/src/satviz/VideoController.cpp

```diff
@@ -37,17 +46,21 @@ void VideoController::processEvent(sf::Event &event) {
     camera.setZoom(camera.getZoom() * factor);
   }
   if (event.type == sf::Event::KeyPressed) {
+    const float SPEED = 200.0f;
```

Comment:
Ist die Vertausendfachung der Geschwindigkeit hier gewollt?

Reply:
Ja. Der PR fixt gleichzeitig einen Bug in toMatrix(), welcher dafür gesorgt hatte, dass die Größenordnungen nicht gestimmt hatten.